providers/cassandra/shedlock-provider-cassandra/src/main/java/net/javacrumbs/shedlock/provider/cassandra/CassandraLockProvider.java

```diff
@@ -42,6 +42,10 @@ public CassandraLockProvider(@NonNull CqlSession cqlSession) {
     public CassandraLockProvider(@NonNull CqlSession cqlSession, @NonNull String table, @NonNull ConsistencyLevel consistencyLevel) {
         this(Configuration.builder().withCqlSession(cqlSession).withTableName(table).withConsistencyLevel(consistencyLevel).build());
     }
+    
+    public CassandraLockProvider(@NonNull CqlSession cqlSession, @NonNull String table, @NonNull ConsistencyLevel consistencyLevel, @NonNull ConsistencyLevel serialConsistencyLevel) {
```

Comment:
Please do not add this constructor, the one with `Configuration` should be sufficient.

Reply:
I added this for convenience and awareness so that people are aware that for CAS they have an explicit option to specify consistency level. If you still want me to remove, I will surely remove it.

Reply:
Yes please

Reply:
Done !